duckduckgo/duckduckgo-privacy-extension
# Remove nudge

**Reviewer:** @jdorweiler 

## Description:
Removing all code related to banner nudge experiment


## Steps to test this PR:
- Visit https://duckduckgo.com?natb=v123-4rz
- Build + install extension with this tab open
    - Should see post-install page
- Visit https://google.com
	- Should not see nudge banner
- Visit https://duckduckgo.com/?va=x&t=hx&atb=v123-1&natb=v226-6xx&cp=atbhc&npi=1
- Install extension with this tab open
    - Should NOT see post-install page
	- Should redirect to step2 of multi-step onboarding experiment


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
